inference-engine/src/readers/ir_reader/ie_ir_reader.cpp

```diff
@@ -27,29 +28,29 @@ bool IRReader::supportModel(std::istream& model) const {
     const int header_size = 128;
     std::string header(header_size, ' ');
     model.read(&header[0], header_size);
+    model.seekg(0, model.beg);
+
+    // Find '<net ' substring and version in the .xml file
+    // Regular expression format:
+    //  (Opional comments or spaces)
+    //      and (<net name="string" version="number"> or <net version="number" name="string">)
+    //      and (Opional comments or spaces)
+    std::regex regex(R"rs((?:(?:<!--[\s\S]*-->)|\s)*(?:<[nN]et\s*name\s*=\s*"(?:[^<&"])*"\s*version\s*=\s*"\s*(\d*)\s*"\s*>|<[nN]et\s*version\s*=\s*"\s*(\d*)\s*"\s*name\s*=\s*"(?:[^<&"])*"\s*>)(?:(?:<!--[\s\S]*-->)|\s)*)rs",
```

Comment:
regex is not supported on CentOS with obsolete compiler. Can we extract the version part of IR until the `>` (it's a `<net ... version="10">`) and parse via pugixml?

Reply:
Fixed